zomes/rea_agent/integrity_zome/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+/**
+ * Holo-REA agent integrity zome for API definition
+ *
+ * Defines the top-level zome configuration needed by Holochain's build system
+ * to bundle the app. This basically involves wiring up the helper methods from the
+ * related `_lib` module into a packaged zome WASM binary.
+ *
+ * @package Holo-REA
+ */
+
+pub use hc_zome_rea_agent_storage::EntryTypes;
+pub use integrity_types::LinkTypes;
```

Comment:
does this work? or do the procedural macros (`#[hdk_entry_defs]` etc) need to be invoked in this crate? @pospi @Connoropolous

Reply:
I suppose a different approach would be to define the `EntryTypes` enum in this file, and implement the `From<EntryStorage>` trait directly in here.

Reply:
FYI @pospi 
Wesley and I jammed on this today and came up with some strategies